packages/p2p/src/stores/general-store.js

```diff
@@ -374,6 +379,23 @@ export default class GeneralStore extends BaseStore {
         this.order_table_type = order_table_type;
     }
 
+    @action.bound
+    setP2PConfig() {
```

Comment:
Where are we calling this method?

Reply:
It will be called in P2P create and edit add sections